src/VirtoCommerce.Platform.Core/PlatformOptions.cs

```diff
@@ -27,7 +27,10 @@ public class PlatformOptions
         //Local path to private key for signing license
         public string LicensePrivateKeyPath { get; set; }
 
-        //Url for discovery sample data for initial installation
+        //Local path for countries list
+        public string CountriesFilePath { get; set; } = "localization/common/countries.json";
```

Comment:
I mean this [path](https://github.com/VirtoCommerce/vc-platform/tree/master/src/VirtoCommerce.Platform.Web/wwwroot/Localizations). 
```suggestion
        public string CountriesFilePath { get; set; } = "wwwroot/localizations/common/countries.json";
```

Reply:
wwwroot/Localizations is dedicated to actual localizations. Can @tatarincev suggest something else?